mkdocs/mkdocs.yml

```diff
@@ -64,7 +75,10 @@ extra:
       link: https://github.com/iter8-tools/iter8
     - icon: fontawesome/brands/slack
       link: https://join.slack.com/t/iter8-tools/shared_invite/zt-awl2se8i-L0pZCpuHntpPejxzLicbmw
-
+  
+  iter8:
+    install_version: v0.7.19
```

Comment:
@kalantar This is where we can change the install version in the `mkdocs.yml`.

Reply:
Need to be updated already.